docs/src/start/connect-azure.md

```diff
@@ -224,9 +224,9 @@ Alternatively, post your own custom messages on `az/messages/events/#` topic:
 tedge mqtt pub az/messages/events/ '{"text": "My message"}'
 ```
 
-To view the messages that were sent from the device to the cloud, follow this [document](https://docs.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-cli#create-and-monitor-a-device).
+To view the messages that were sent from the device to the cloud, follow this [document](https://learn.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-cli).
```

Comment:
The hash is still valid.
https://learn.microsoft.com/en-us/azure/iot-hub/quickstart-send-telemetry-cli#create-and-monitor-a-device

Reply:
Resolved by 98f0b059b593f5ee8d827baf86890ee5a487bd89